src/ring_theory/power_series/basic.lean

```diff
@@ -1132,6 +1162,24 @@ mv_power_series.mul_inv_of_unit φ u $ h
 
 end ring
 
+section comm_ring
+variables (A : Type*) [comm_ring A]
+
+/-- The ring homomorphism taking a power series `f(X)` to `f(-X)`. -/
+noncomputable def eval_neg_hom : power_series A →+* power_series A :=
+eval_mul_hom (-1 : A)
+
+@[simp] lemma eval_mul_hom_neg_one_X : eval_mul_hom (-1 : A) X = -X :=
+begin
+  ext, simp only [linear_map.map_neg, coeff_eval_mul_hom, coeff_X],
+  split_ifs with h; simp [h]
+end
+
+@[simp] lemma eval_neg_hom_X : eval_neg_hom A X = -X :=
+eval_mul_hom_neg_one_X A
```

Comment:
nit: I think this maybe reads better with the order swapped, to keep the definition and its lemma together

```suggestion
@[simp] lemma eval_mul_hom_neg_one_X : eval_mul_hom (-1 : A) X = -X :=
begin
  ext, simp only [linear_map.map_neg, coeff_eval_mul_hom, coeff_X],
  split_ifs with h; simp [h]
end

/-- The ring homomorphism taking a power series `f(X)` to `f(-X)`. -/
noncomputable def eval_neg_hom : power_series A →+* power_series A :=
eval_mul_hom (-1 : A)

@[simp] lemma eval_neg_hom_X : eval_neg_hom A X = -X :=
eval_mul_hom_neg_one_X A
```